nodes/object/object_name_filter.py

```diff
@@ -0,0 +1,54 @@
+import bpy
+from ... events import executionCodeChanged
+from ... base_types.node import AnimationNode
+
+class ObjectNameFilterNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_ObjectNameFilterNode"
+    bl_label = "Object Name Filter"
+    
+    def checkedPropertiesChanged(self, context):
+        executionCodeChanged()
```

Comment:
when the update function only contains *executionCodeChanged* you don't need this extra update function. Just call executionCodeChanged directly from the property